### DISCUSSION TITLE
Image not loading on website

### DISCUSSION THREAD

**Discussion by kasjfksj** [alshedivat/al-folio#2656] [Q&A]
I'm trying to display an image on my post, adding <img src = "./img/35001724304205_.pic.jpg"> in my post.md.

The image does display when I view it from code on github
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/d2e55fab-fbe0-4342-b048-7b9ab0a0ec17">

However, when I go to my website, the image is not loaded
<img width="885" alt="image" src="https://github.com/user-attachments/assets/08df2494-806e-4dee-9295-fec92236570f">

I don't understand how this can happen. Someone please helps me out!

**Comment by george-gca** [comment]
What is the link to your site and repository?
